gen/PathPlanner.yml

```diff
@@ -7,5 +7,22 @@ classes:
     methods:
       loadPath:
         overloads:
-          std::string, units::meters_per_second_t, units::meters_per_second_squared_t, bool:
-          std::string, units::meters_per_second_t, units::meters_per_second_squared_t:
+          std::string&, PathConstraints, bool:
+          std::string&, units::meters_per_second_t, units::meters_per_second_squared_t, bool:
+      loadPathGroup:
+        overloads:
+          std::string&, std::initializer_list<PathConstraints>, bool:
+            ignore: true
+          std::string&, units::meters_per_second_t, units::meters_per_second_squared_t, bool:
+      generatePath:
+        overloads:
+          PathConstraints, bool, PathPoint, PathPoint, std::initializer_list<PathPoint>:
+            ignore: true
+          ? units::meters_per_second_t, units::meters_per_second_squared_t, bool,
+            PathPoint, PathPoint, std::initializer_list<PathPoint>
+          : ignore: true
+          PathConstraints, PathPoint, PathPoint, std::initializer_list<PathPoint>:
+            ignore: true
+          units::meters_per_second_t, units::meters_per_second_squared_t, PathPoint, PathPoint, std::initializer_list<PathPoint>:
+            ignore: true
```

Comment:
Yes, in 2023.2.0 these really all take `initializer_list`. `vector` overloads should be available in the next release:

- https://github.com/mjansen4857/pathplanner/pull/178#issuecomment-1367091321